openedx/core/lib/course_tabs/tabs.py

```diff
@@ -1,13 +1,15 @@
 """
 Implement CourseTab
 """
-from abc import ABCMeta
```

Comment:
I find it odd to have files called tabs.py and course_tabs.py, especially as CourseTab is not in course_tabs.py! Can these two files be consolidated now?